### ISSUE TITLE
The Destroyer From Jail

### ISSUE THREAD

**Issue by arnoldsecret** [finalburnneo/FBNeo#134]
http://adb.arcadeitalia.net/dettaglio_mame.php?game_name=dfjail

Runs on System 16 hardware

**Comment by fastpop72** [comment]
Wow! Seems cool!

**Comment by barbudreadmon** [comment]
>the other 3 have been supported by fbneo

This is irrelevant since dfjail uses different (and unsupported) components.

**Comment by wn2000** [comment]
Understood. I'm still hopeful as it seems from mame's code there's not too much special handling for dfjail.

**Comment by grant2258** [comment]
@wn2000 there doesnt seem to be any extra components from the other Korean games. I think this might be worth adding to plus will need to wait until the festive period, works a bit of  nightmare at the moment. Ill let you know if I bump into any issues with missing components though pretty sure FBNeo has the segaic I dont see what else could be missing. Its actually a pretty decent game played it for the first time today.

**Comment by grant2258** [comment]
Managed to get the game up and running on plus just need to do the palette handling and fix the sound up as the speech is missing at the moment. Its should be doable on this core if any of the devs are interested in adding it. Its quiet a nice little game kinda remind me of narc a little bit quiet a decent creation for a bootleg.

kind regards

**Comment by barbudreadmon** [comment]
To clarify about the unsupported components : except if the game is wrongly flagged on arcade italia, it uses analog circuitry for sound, which we don't emulate, and while samples can sometime be used as a substitute, i have no idea if that's feasible here.

Emulating this game with bad/incomplete sound has always been an option, and probably the reason i left this issue open back in 2019 despite the fact we don't accept requests for not-yet-emulated hardware.

**Comment by grant2258** [comment]
Dont see implementing the dac as being an issue on plus data can easily be manipulated to our dac data format if it differs , the interrupts need redone for the new code as well. Its not an overly complicated task on plus side anyway since a dac  converts a digital signal into an analog buffered or unbuffered voltage or current nothing more, the ad7533 specs are available it wouldn't be a massive task to adjust the voltage tables if needed. Just need motivation and free time to do it.

**Comment by barbudreadmon** [comment]
To clarify, i was talking about the netlist sound device, not the dac.

**Comment by grant2258** [comment]
> , i was talking about the netlist sound device, not the dac.

@barbudreadmon not sure if you winding me up but I will go with the thought your not. I would imagine all system16b if not most have the netlist device and they are still added to fbneo. Bayroute, cotton, ddux ect. Anyway all thats needed for the speech is the dac on the plus side.

**Comment by barbudreadmon** [comment]
You are right, i had only looked at the other segas16b korean bootlegs and noticed they didn't have a netlist sound device, but it appears the normal segas16b do have one and it refers to some kind of amplifier.

**Comment by grant2258** [comment]
I finished it up in plus speech as well so its all good can be added if someone is motivated. Im not really familiar with the fbneo code bade else I would attempt it.

**Comment by barbudreadmon** [comment]
done
